canonical/maas-ui
# chore: Refactor MachineDetails to use PageContent MAASENG-1820

## Done

- Refactored MachineDetails and MachineHeader to use PageContent
- Renamed MachineHeaderForms to MachineForms

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to `/machines` and click a machine to go to its details
- Ensure clicking each action opens the correct form and it displays correctly

## Fixes

Fixes [MAASENG-1820](https://warthogs.atlassian.net/browse/MAASENG-1820)


[MAASENG-1820]: https://warthogs.atlassian.net/browse/MAASENG-1820?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
